classes/db/DbQuery.php

```diff
@@ -93,7 +93,13 @@ public function select($fields)
     public function from($table, $alias = null)
     {
         if (!empty($table)) {
-            $this->query['from'][] = '`' . _DB_PREFIX_ . $table . '`' . ($alias ? ' ' . $alias : '');
+            if ($table instanceof DbQuery) {
+                $query = '(' . $table->build() . ')';
```

Comment:
@PierreRambaud in your example you have: 

`$selectWithSubQuery->from($subQuery->build(), 'p');`

but you're calling `->build()` again, am I missing something or it should be just `$table` here or in the argument?

Reply:
My bad, forget to edit my description :sweat_smile: